README.md

```diff
@@ -5,4 +5,74 @@ This repository contains instructions for benchmarking ewasm contracts and stand
 ```
 ewasm/          - contains benchmarks and tests for ewasm contracts in ewasm engines.
 wasm/           - contains benchmarks for wasm modules in standalone wasm engines.
+wasm-engines/   - contains benchmarks for wasm modules comparing wasm engines.
 ```
+
+## Run benchmarks
+
+### Wasm Engines Benchmarks
+
+Build the docker image:
+
+    $ cd wasm-engines
+    $ docker build . -t wasm-engines
+    
+Run the docker container:
+
+    $ cd ..
+    $ docker run --privileged -v $(pwd)/wasm-engines/wasmfiles:/wasmfiles -v $(pwd)/benchmark_results_data:/benchmark_results_data --security-opt seccomp=$(pwd)/wasm-engines/dockerseccompprofile.json -it wasm-engines /bin/bash
+
+Build the wasm binaries and execute benchmarks:
+
+    root@docker# ./bench_wasm_and_native.sh
+    
+### Scout Benchmarks
+
+Build the docker image:
+
+    $ cd scout-engines
+    $ docker build . -t scout-engines
+    
+Run the docker container:
+
+    $ cd ..
+    $ docker run --privileged -v $(pwd)/benchmark_results_data:/benchmark_results_data -it scout-engines /bin/bash
+
+Run benchmarks
+
+    root@docker# python3 scout_bignum_bench.py
+
+## Generate charts using jupyter notebooks
+
+The simplest way to install jupyter notebooks and the dependencies needed to generate the chart is by using Conda:
+
+1. Download the miniconda installer from: https://docs.conda.io/en/latest/miniconda.html
+2. Execute the installer
+```
+	$ bash Miniconda3-latest-Linux-x86_64.sh
```

Comment:
At least on my Ubuntu 18.04 machine, it was pretty easy to install all of the deps for jupyter using Pip.  Could consider wrapping the jupyter notebook stuff in a Docker container to make things easier.

Probably not extremely high-priority at this point.